test/DynamoCoreWpfTests/WebRequestTests.cs

```diff
@@ -12,11 +12,12 @@
 using NUnit.Framework;
 
 using WebRequest = CoreNodeModels.WebRequest;
+using Dynamo.Graph.Nodes.ZeroTouch;
 
 namespace DynamoCoreWpfTests
 {
     [TestFixture]
-    public class WebRequestTests : SystemTestBase
+    public class PeriodicUpdateTest : SystemTestBase
```

Comment:
There is a test case called WebRequest. It looks not quite proper to change the name.